static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -86,6 +86,17 @@ define([
                 modClass: 'end-article',
                 insertMethod: insertBottomOfArticle
             },
+            usBriefing: {
+                listId: '',
```

Comment:
Hi @crifmulholland - can you provide me with the list id, campaign code, headline, description and success description please?